test/e2e/fixtures/README.md

```diff
@@ -7,30 +7,17 @@ one thing:
 
 ## Generating fixture data
 
-Fixture data can be generated by loading the unpacked extension, inspecting the background context,
-dumping `chrome.storage.local`, and using [`copy`][1] to copy it to the clipboard, and manually pasting the contents into
-a file on disk.
+Fixture data can be generated by following these steps:
 
-```js
-// Step 1:
-chrome.storage.local.get(null, (x) => console.log(x))
-// Should print something like:
-// > temp1
-// > {data: {…}, meta: {…}}
+1. Load the unpacked extension in development or test mode
+2. Inspecting the background context of the extension
+3. Call `metamaskGetState`, then call [`copy`][1] on the results
 
-// Step 2:
-// Store the value as a global via the 'Store as global variable' option in context menu (shown below)
+You can then paste the contents directly in your fixture file.
 
-// Step 3:
-copy(temp1)
+```js
+metamaskGetState().then((state) => copy(state)).catch(console.error)
```

Comment:
I believe we can use a top-level `await` in the console now, yeah?

Reply:
Ah right, good point.